public/js/util/tests/sources-tree.js

```diff
@@ -7,6 +7,109 @@ const {
 } = require("../sources-tree.js");
 
 describe("sources-tree", () => {
+  const abcSource = Map({
+    url: "http://example.com/a/b/c.js",
+    actor: "actor1"
+  });
+  const abcdeSource = Map({
+    url: "http://example.com/a/b/c/d/e.js",
+    actor: "actor2"
+  });
+  const abxSource = Map({
+    url: "http://example.com/a/b/x.js",
+    actor: "actor3"
+  });
+
+  it("can collapse a single source", () => {
```

Comment:
do you mind moving these collapse tests below the tests that were already in here? these seem like tests for the more advanced feature of collapsing and should be after the tests of the basic API